### ISSUE TITLE
`bundled-sqlcipher-vendored-openssl` doesn't compile on windows

### ISSUE THREAD

**Issue by douloiha** [rusqlite/rusqlite#1025]
When using the `bundled-sqlcipher-vendored-openssl` feature, it produces the following error:

```
could not find native static library `crypto`, perhaps an -L flag is missing?
```

I guess this is because in the `libsqlite3-sys/build.rs` file, line 184 doesn't distinguish windows and linux crypto library. On windows, the library is named libcrypto (and not crypto).

**Comment by thomcc** [comment]
Yeah, I think it's called something else on windows (libeay32 i think). Anyway, it's known to be unsupported (the README calls bundled-sqlcipher "more or less Unix only at the moment"). But honestly I had kinda intended to get back to it at some point and didnt, 

I'll see if I can figure something out, since I don't really think surprise breakage on windows is ok, but... hrm.
